config/clients/js/template/example/example1/package.json.mustache

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "example1",
+  "private": "true",
+  "version": "1.0.0",
+  "description": "A bare bones example of using the OpenFGA SDK",
+  "author": "OpenFGA",
```

Comment:
do you need `"type" = "module"` for `.mjs` support?

Reply:
Not if we're using  the`.mjs` extension, if we wanted to use `.js` then we would have to yeah